Utilities/Analyzer.cc

```diff
@@ -71,84 +61,11 @@ class Analyzer{
     cout << "Output will be saved to " << ofname << endl;
     ofile->cd();
     t = new TTree("t","EventTree");
-    BookBranches();
-    evtCounter = new TH1F("EventCounter","Event Counter", 10, 0.5, 10.5);
   }
 
-  int ReadEvent(Long64_t ievt) {
+  void ReadEvent(Long64_t ievt) {
     iEvent = ievt;
-    evtCounter->Fill("All",1);
     r->ReadEvent(ievt);
-    if (conf->PrintProgress) progress->Print(ievt);
```

Comment:
The entire selection logic has been moved into the NanoAODReader.

Reply:
That's fine. It was put here in the past before I introduced config. The PrintProgress switch can be passed only once from the main code to Analyzer. Now every config is in Configs, so it doesn't matter where you put it.